src/Gl/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.9)
 project(Gl)
 
-find_package(OpenGL REQUIRED COMPONENTS EGL)
+find_package(OpenGL COMPONENTS EGL)
```

Comment:
Removing REQUIRED might break other builds. In thery the RaspberryPi.cmake could populate the correct variables to avoid this find_packages being triggered.